portal.h

```diff
@@ -62,6 +57,62 @@ void setSSID(String ssid){
   return;
 }
 
+void confirmOrTypeSSID(){
+  DISP.fillScreen(BLACK);
+  DISP.setSwapBytes(true);
+  DISP.setTextSize(MEDIUM_TEXT);
+  DISP.setTextColor(TFT_RED, BGCOLOR);
+  DISP.setCursor(0, 0);
+  DISP.println("WiFi SSID");
+  DISP.setTextSize(TINY_TEXT);
+  DISP.setTextColor(FGCOLOR, BGCOLOR);
+  DISP.println(TYPE_SSID_TEXT);
+  DISP.setTextSize(SMALL_TEXT);
+  uint8_t ssidTextCursorY = DISP.getCursorY();
+  String currentSSID = String(apSsidName.c_str());
+  DISP.printf("%s", currentSSID.c_str());
+  bool ssid_ok = false;
+
+  while(!ssid_ok){
```

Comment:
I like the concept, but I'd like to set it up so NEMO Portal launches the portal right when you select the menu option. Could we move the SSID Modification flow so when a special button is pressed on a cardputer, we can alter the SSID?

Reply:
Hmmm that is a interesting point.
Thinking about user experience, in my opinion, the SSID definition step wouldn't impact the UX.

In fact, I think the current flow is very clean because the previously used SSID will be populated on the screen, so if the user just wants to launch the NEMO Portal, it can simply press Enter and the previous SSID will be used.
What do you think?